.github/workflows/stale.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Stale issue handler'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 7 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4.0.0
+        id: stale
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 300 days with no activity. Remove stale label or comment or this will be closed in 30 days'
+          stale-pr-message: 'This PR is stale because it has been open for 300 days with no activity. Remove stale label or comment or this will be closed in 30 days'
```

Comment:
NIT: Suggestion on the last sentence: 'This PR will be closed in 30 days.  Please remove the "Stale" label or comment to avoid closure with no action.'